public/app/features/logs/components/panel/processing.ts

```diff
@@ -90,7 +99,8 @@ export class LogListModel implements LogRowModel {
     this._grammar = grammar;
     this.timestamp = dateTimeFormat(log.timeEpochMs, {
       timeZone,
-      defaultWithMS: true,
+      // YYYY-MM-DD HH:mm:ss.SSS
+      format: systemDateFormats.fullDateMS,
```

Comment:
Here we're just making sure we control the output format, in case `dateTimeFormat()` internally changes and we get an erroneous timestamp with appended nanoseconds.